react/week4/github-users-search-app/src/components/searchUsersApi.js

```diff
@@ -0,0 +1,8 @@
+const searchUsersApi = async (searchQuery) => {
+  const API = await fetch(
+    `https://api.github.com/search/users?q=${searchQuery}`
+  ).then((response) => response.json());
+  return API;
+};
+
+export { searchUsersApi };
```

Comment:
When there’s only one default in a file like here then it’s good practice to export it as the “default” function from that file like so:

"export default searchUsersApi" instead of export { searchUsersApi }

You can only have 1 default when exporting per file while you can have many { explicit } exports from a file

When you import in the other file you simply import "import searchUsersApi from '../file-name.js" (without the curly brackets)